src/Common/Core/Impl/IO/Exports.cs

```diff
@@ -2,12 +2,8 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.ComponentModel.Composition;
-using Microsoft.Common.Core.IO;
-using Microsoft.VisualStudio.ProjectSystem.Utilities;
 
-namespace Microsoft.VisualStudio.R.Package.ProjectSystem {
-
-    [AppliesTo("RTools")]
+namespace Microsoft.Common.Core.IO {
```

Comment:
I'll undo this one but there is a bunch of missing MEF imports getting thrown in interactive tests (including this one), need to clean them up eventually. Luckily interactive tests don't rely on those, but that is just so far...